client/transactions/list/style.scss

```diff
@@ -123,6 +123,8 @@ $gap-small: 12px;
 
 			.woocommerce-select-control__control {
 				height: 38px;
+				overflow: scroll;
+				align-items: flex-start;
```

Comment:
Although it is a minor fix, this change isn't behind the feature flag.

Perhaps it would help communicate this change if we had 2 changelog entries:
1. Fix for view report behind feature flag.
2. Fix for Transaction List styles

Alternatively, we could move this change to a separate issue/PR. This may be overkill, but it may help us step back and look at the problem/solution separate to the Payment Activity Card.

Reply:
As an alternative UX to `overflow: scroll`, I would consider letting the search box to increase in height when necessary, example:

![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/d69c10ff-3234-4f44-a50e-c0a2135ca003)

<img width="361" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/4b101556-754c-4c0c-8d9d-293ffc8e808e">

_(avoiding the `!important` if possible 😉)_

Reply:
>Although it is a minor fix, this change isn't behind the feature flag.

I had completely missed this! Thanks for sharing!. 

I would favor creating a separate issue + PR. It would be tiny but keep things clear.

Reply:
@nagpai Thanks for rising it and @Jinksi thanks for coming up with the fix. I feel like the increase in height is the best we can do now. For the narrower (mobile) screens - I don't know if this is possible – but I'd:
1. put the applied filter (tags) in one line (could they be horizontally scrollable?) 
2. have the `x` cross sign for the user to clear the filters visible 
2.  have the next (2nd) line empty so that the user can search if they want

In a nutshell, I'd prioritise the user's ability to search over seeing all the applied filters.

Ugh, does it make sense?

Reply:
@rogermattic  Thanks, Magda! I will add this guidance in the new issue I create for improving the mobile UI 🙏 !

Reply:
I have created https://github.com/Automattic/woocommerce-payments/issues/8781 to track this change here!